src/SignalR/clients/cpp/test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -331,17 +236,6 @@ TEST(hub_connection_tests, connection_id_start_stop_start_reconnect)
     ASSERT_NE(hub_conn->get_connection_id(), connection_id);
 
     connection_id = hub_conn->get_connection_id();
-
-    try
-    {
-        hub_conn->send(U("forceReconnect")).get();
```

Comment:
It looks like nothing calls HubConnection.ForceReconnect anymore in signalrclient-testhost. Should it be removed?